core/email.py

```diff
@@ -14,7 +14,7 @@
 
 from threepio import logger
 
-from atmosphere import settings
```

Comment:
According to the Travis tests, line 154 of this file is causing the error because of `settings._get_method_for_string` which is a function defined [here](https://github.com/cyverse/atmosphere/blob/ce15e0f7b77394e37470af5ff959ca2e7679e73e/atmosphere/settings/__init__.py#L540). I think it can be replaced by just changing the line to `lookup_fn = globals()[lookup_fn_str]`

Reply:
I made this change and ran the tests in my dev environment and the tests passed.

The method is used again on [line 166 of the same file](https://github.com/cyverse/atmosphere/blob/ce15e0f7b77394e37470af5ff959ca2e7679e73e/core/email.py#L166)